packages/app/src/providers/mock/mock-api.ts

```diff
@@ -132,43 +136,6 @@ export class MockApi extends DataProviderApi {
   }
 
   public override async getSearchablePaths(path: string): Promise<string[]> {
-    return this.getEntityPaths(path);
-  }
-
-  private getEntityPaths(entityPath: string): string[] {
```

Comment:
Unrelated: I moved a couple of methods out of the `MockApi` class and turned them into utilities instead.